gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -10,19 +10,16 @@
         <div id="device-general" class="tab-pane in active">
           <div ng-if="!editingDeviceDefaults()" class="form-group" ng-class="{'has-error': deviceEditor.deviceID.$invalid && deviceEditor.deviceID.$dirty}" ng-init="loadFormIntoScope(deviceEditor)">
             <label translate for="deviceID">Device ID</label>
-            <div ng-if="!editingDeviceExisting()">
```

Comment:
It's not the same, but isn't using an `input` tag (with a readonly attribute) here more appropriate than a `div`? That's the background of this change. This still is an input field after all (even if non-editable).

Reply:
I dunno, didn't design it in the first place.  There were some discussions about it in #7653 and in the end we decided to keep it as is.  The select-on-click behavior is the main difference I suppose.

Reply:
See the other comment, I now think it's better to switch to the original "well" version in your specific case.

Reply:
I have switched to the "well" version now, but I've also grouped everything together instead of having two separate sections. This way it's possible to reuse the same "Show QR" button. Please let me know what you think of it.